config_center/nacos/impl.go

```diff
@@ -36,7 +36,10 @@ import (
 	"github.com/apache/dubbo-go/config_center/parser"
 )
 
-const nacosClientName = "nacos config_center"
+const (
+	nacosClientName = "nacos config_center"
+	maxKeysNum      = 9999
```

Comment:
well done Ming

Reply:
9999，are you sure？my dear Mind.

Reply:
One key for one application. 9999 is enough...